tests/Acceptance/ApplicationTest.php

```diff
@@ -292,6 +297,6 @@ private function assertSimpleScript(int $exitCode, string $envName): void
         $this->assertStringContainsString('All commands successfully executed!', MockWriter::$content);
         $this->assertStringContainsString('Duration:', MockWriter::$content);
 
-        self::assertStringEqualsFile(__DIR__ . '/_app/result.txt', $envName);
+        $this->assertStringEqualsFile(__DIR__ . '/_app/result.txt', $envName);
```

Comment:
Previous implementation was correct.
```suggestion
        self::assertStringEqualsFile(__DIR__ . '/_app/result.txt', $envName);
```